.github/workflows/source-web-tool-scripts.yaml

```diff
@@ -37,59 +21,24 @@ jobs:
           repository: 2DegreesInvesting/PACTA_analysis
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: PACTA_analysis
-
       - name: Checkout pacta-data
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/pacta-data
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: pacta-data
-
       - name: Checkout create_interactive_report
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/create_interactive_report
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: create_interactive_report
-
       - name: Checkout StressTestingModelDev
         uses: actions/checkout@v2
         with:
           repository: 2DegreesInvesting/StressTestingModelDev
           token: ${{ secrets.MAURO_PAT_FOR_2DII }}
           path: StressTestingModelDev
-
-      - uses: r-lib/actions/setup-pandoc@v1
-      - uses: r-lib/actions/setup-tinytex@v1
-      - uses: r-lib/actions/setup-r@v1
-        with:
-          r-version: ${{ matrix.config.r }}
-
-      - name: Install system dependency of usethis >= 2.0.0
-        if: runner.os == 'Linux'
-        run: |
-          sudo apt install libgit2-dev
-
-      - name: Install dependencies from CRAN
-        run: |
-          # TODO Install elsewhere? Why is it not detected by renv?
-          install.packages("forcats")
-          install.packages("rmarkdown")
-          install.packages("knitr")
-          install.packages("renv")
-          deps <- unique(renv::dependencies("PACTA_analysis")$Package)
-          exclude <- c(
-            "R",
-            "utils",
-            "base",
-            "tools",
-            "create_interactive_report",
-            "PACTA.analysis",
```

Comment:
PACTA_analysis?

Reply:
CRAN does not allow package names to contain `_`, so although the repo is named `PACTA_analysis`, the pkg infrastructure, e.g. the DESCRIPTION file, refer to it as `PACTA.analysis`